docs/snapshotting/snapshot-support.md

```diff
@@ -118,7 +118,7 @@ The baseline for snapshot resume latency target on Intel is under **8ms** with
   For recommendations related to guest network connectivity for clones please
   see [Network connectivity for clones](network-for-clones.md).
 - Vsock device does not have full snapshotting support.
-  Please see [Vsock device limitations](#vsock-device-limitations).
+  Please see [Vsock device limitations](#vsock-device-limitation).
```

Comment:
I suggest to modify all occurences to use singular (i.e "Vsock device limitation")